Content/LeagueSandbox-Scripts/Spells/Items/Passives/Doran's Shield.cs

```diff
@@ -0,0 +1,29 @@
+﻿using GameServerCore.Domain.GameObjects;
+using GameServerCore.Domain.GameObjects.Spell;
+using GameServerCore.Domain.GameObjects.Spell.Missile;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+using System.Numerics;
+using GameServerCore.Scripting.CSharp;
+using LeagueSandbox.GameServer.GameObjects.Stats;
+using LeagueSandbox.GameServer.API;
+
+namespace ItemPassives
+{
+    public class ItemID_1054 : IItemScript
+    {
+        public IStatsModifier StatsModifier { get; private set; } = new StatsModifier();
+
+        public void OnActivate(IObjAiBase owner)
+        {
+            StatsModifier.HealthRegeneration.BaseBonus += 1.2f;
```

Comment:
+6 health regen per 5 seconds

Reply:
Yeah, 6 regen per 5 seconds translates to 1.2 per second (6/5)